sql/Draenor/2016_03_16_00_world.sql

```diff
@@ -0,0 +1,23 @@
+--- Bloodmaul Slag Mines
```

Comment:
You have to make a correction on Line 1.
You have a syntax error.
Line 1: - - - ( you should have only 2 - - not 3 - - - )
You are not able to import the sql on database cause of line 1. 
This is the correct syntax: -- Bloodmaul Slag Mines